### DISCUSSION TITLE
Decreasing Git LFS storage to restore access

### DISCUSSION THREAD

**Discussion by gregtom6** [git-lfs/git-lfs#5337] [Q&A]
Hello,

I received this message today, when I tried to push:
"This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access."

Now I try to find a solution, which avoids paying. 
I don't have experience in Git LFS, but I think the problem is related to that I used one very big asset (4-5 files) and tracked them in Git LFS previously. 
I deleted those 4-5 files from the repo, but the size of .git/lfs/objects (is this the folder which size needs less than 1 GB?) is still the same. 

[https://docs.github.com/en/repositories/working-with-files/managing-large-files/removing-files-from-git-large-file-storage#git-lfs-objects-in-your-repository](url)
In here they mention "To remove Git LFS objects from a repository, delete and recreate the repository." Isn't there a way to do this without deleting the repo?
Isn't there a way to somehow leave the repo untouched, or modify or delete some of the previous commits? My repo would be seen by recruiters and I don't want to break the urls for them, it would be good if they would at least see the commits (even if some of them are not valid anymore), but the latest commits would surely work for them.

**Comment by bk2204** [comment]
Hey,

The open source Git LFS client doesn't provide a way to delete objects on the server.  It's simply not in the protocol, so there's no way to ask the server to delete objects you don't want anymore.

However, having said that, on GitHub, you can ask Support to delete certain objects for you if you need to delete only a few.
